Vagrantfile

```diff
@@ -0,0 +1,27 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+    config.vm.provider "libvirt" do |libvirt, override|
+        libvirt.memory = 2048
+        libvirt.cpus = 3
+    end
+  config.vm.synced_folder ".", "/vagrant", type: "rsync",
+    rsync__exclude: ".tmp*"
+
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "fedora/24-cloud-base"
```

Comment:
We actually have Vagrant boxes generated for CAHC (both continuous and alpha) btw, and I switched to using CAHC Alpha in https://github.com/projectatomic/rpm-ostree/pull/502